app/views/teacher_interface/application_forms/show/_submitted.html.erb

```diff
@@ -1,12 +1,13 @@
 <% further_information_submitted = view_object.further_information_request&.received? %>
 <% qualification_consent_submitted = view_object.qualification_consent_submitted? %>
+<% teaching_authority_provides_written_statement = view_object.application_form.region.teaching_authority_provides_written_statement? %>
```

Comment:
Can all the above above be put into the `TeacherInterface::ApplicationFormViewObject` as methods instead of here?